### ISSUE TITLE
No nps_psx?

### ISSUE THREAD

**Issue by zemmyang** [sigmaboy/nopaystation_scripts#8]
Is there a way to also get PSX games using these scripts? Or am I missing something?

**Comment by sigmaboy** [comment]
You're correct. This needs to be implemented. Please check at the code if you're able to.
